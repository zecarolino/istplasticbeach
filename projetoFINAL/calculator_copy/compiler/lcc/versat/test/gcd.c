/* maior divisor comum entre dois numeros inteiros positivos */
int a = 144;
int b = 42;
/* expect 6 */

int main(int argc, char* *argv, char* *envp) {
  do {
    if (a > b)
      a = a - b;
    else
      b = b - a;
  } while (a != b);
  return a;
}
