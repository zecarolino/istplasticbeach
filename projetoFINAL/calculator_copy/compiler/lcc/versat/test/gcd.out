VCD info: dumpfile xtop.vcd opened for output.
6
Catched Trap at time  907 clock cycles:
